.github/workflows/bench.yml

```diff
@@ -111,6 +127,12 @@ jobs:
             failure: 'false'
             result: ${{ env.BENCH_REPORT_FILE }}
 
+      - name: Stop Instance
```

Comment:
I belive it should happen after the `strategy` matrix. Otherwise I'll stop the instance after each nodejs version

Reply:
you're right, it must be in separate JOB.